src/js/_enqueues/wp/customize/nav-menus.js

```diff
@@ -1574,6 +1576,84 @@
 			};
 		},
 
+		/**
+		 * 
+		 * Set up initial state of accessibility for menu items.
```

Comment:
Make this more explicit: it's setting up the initial state of the screen reader accessibility information for menu items. Setting up the state of accessibility implies it's doing things outside the parameters that it's actually handling.